esprima.js

```diff
@@ -3616,6 +3692,10 @@
         startToken = lookahead;
         token = lookahead;
 
+        if (matchKeyword('yield')) {
```

Comment:
The spec outlines the rules for it, but yield is a contextual keyword/identifier. Basically, anywhere inside a generator it's a keyword, and I *think* also if it's in an arrow inside a generator (on my phone now but would have to confirm in the spec).

Also beware of 'yield' context in function params space:
https://twitter.com/bterlson/status/603016789379846144